README.md

```diff
@@ -1,113 +1,143 @@
+  - Join the #tuf channel on CNCF Slack for real-time discussions and support.
+
+Questions, feedback, and suggestions are welcomed on our mailing list or the CNCF Slack channel.
+
+---
+
+## Security Issues and Bugs
+
+[![Security](https://img.shields.io/badge/Security%20Issues-SECURITY.md-red)](docs/SECURITY.md)
+
+If you encounter any security issues or bugs, please refer to the [security policy](docs/SECURITY.md) for reporting and guidelines.
+
+---
+
+## License
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+This work is dual-licensed and distributed under the (1) MIT License and (2) Apache License, Version 2.0. Please see [LICENSE-MIT](https://github.com/theupdateframework/python-tuf/blob/develop/LICENSE-MIT) and [LICENSE](https://github.com/theupdateframework/python-tuf/blob/develop/LICENSE) for more details.
+
+---
+
+## Acknowledgements
+
+This project is hosted by the Linux Foundation under the Cloud Native Computing Foundation. TUF's early development was managed by members of the [Secure Systems Lab](https://ssl.engineering.nyu.edu/) at [New York University](https://engineering.nyu.edu/). We appreciate the efforts of all [maintainers and emeritus maintainers](https://github.com/theupdateframework/python-tuf/blob/develop/docs/MAINTAINERS.txt), as well as the contributors Konstantin Andrianov, Kairo de Araujo, Ivana Atanasova, Geremy Condra, Zane Fisher, Pankhuri Goyal, Justin Samuel, Tian Tian, Martin Vrachev, and Yuyu Zheng who significantly contributed to TUF's reference implementation. Maintainers and Contributors are governed by the [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+This material is based upon work supported by the National Science Foundation under Grant Nos. CNS-1345049 and CNS-0959138. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation.
+
+---
+
+## <div align="center"> The Update Framework (TUF) is a CNCF graduated project </div>
+
+<div align="center">
+  <a href="https://www.cncf.io/">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://github.com/shubhusion/python-tuf/blob/develop/cncf-white.png">
+      <source media="(prefers-color-scheme: light)" srcset="https://github.com/shubhusion/python-tuf/blob/develop/cncf-black.png">
+      <img src="https://github.com/shubhusion/python-tuf/blob/develop/cncf-black.png" height="150" alt="TUF" style="background-color:white"/>
+    </picture>
+  </a>
+</div>
+
+
+---
+
+[![](https://img.shields.io/badge/Move%20to%20Top-%E2%86%91%20Back%20to%20Top-blue)](#a-framework-for-securing-software-update-systems)
+
```

Comment:
This duplicates functionality people already have in their browsers, except now we have to maintain that functionality. Let's not add things like this.